Formula/mqttx-cli.rb

```diff
@@ -0,0 +1,12 @@
+class MqttxCli < Formula
+  desc "Open source MQTT 5.0 CLI client and MQTT X on the command-line"
```

Comment:
Less duplication and redundant information (since open source is a Homebrew requirement)
```suggestion
  desc "MQTT 5.0 and MQTT X CLI client"
```